pkg/rlptools/wasm_utils.go

```diff
@@ -26,6 +26,23 @@ type GatewayAction struct {
 	Rules []kuadrantv1beta1.Rule `json:"rules,omitempty"`
 }
 
+func DefaultGatewayConfiguration(key client.ObjectKey) []kuadrantv1beta1.Configuration {
+	return []kuadrantv1beta1.Configuration{
+		{
+			Actions: []kuadrantv1beta1.ActionSpecifier{
+				{
+					GenericKey: &kuadrantv1beta1.GenericKeySpec{
+						DescriptorValue: key.String(),
+						// using default value as specified in Envoy spec
+						// https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#config-route-v3-ratelimit-action-generickey
+						DescriptorKey: &[]string{"generic_key"}[0],
```

Comment:
I wonder if there's any difference from the compiler's POV between defining a simple `string` var vs. a fixed length-1 `[]string`. In the end, what we want here is the pointer to a string, right?

Reply:
it's a way to specify an address of a string literal. Let me know if there is a better way